source/_config.yml

```diff
@@ -579,6 +591,18 @@ documents:
     version: "1"
     nav: /_includes/hiera1.html
 
+  /puppetserver/5.1:
+    doc: puppetserver
+    version: "5.1"
+    nav: /puppet/5.2/_puppet_toc.html
+    external_source:
+      repo: git://github.com/puppetlabs/puppetserver.git
+      commit: 
```

Comment:
For the `commit`, get a commit ID from the puppet-server repo after the docs are merged ([PR #1506](https://github.com/puppetlabs/puppetserver/pull/1506)).